### DISCUSSION TITLE
TumblThree v2.13.0 released!

### DISCUSSION THREAD

**Discussion by thomas694** [TumblThreeApp/TumblThree#522] [Announcements]
### TumblThree 2.13.0

- Features
  - Make some tooltips multiline
  - New menu entry in the blog list's context menu to dequeue the selected blogs

- Bug fixes
  - Likes crawler interrupts on error (issue #517)
  - Show error message for occasional error on adding batches of blogs
  - Failed to determine login name (issue #520)
  - Error: Image size couldn't be retrieved (issue #519)
  - Crawlers broken due to page structure change

<hr /><em>This discussion was created from the release <a href='https://github.com/TumblThreeApp/TumblThree/releases/tag/v2.13.0'>TumblThree-v2.13.0</a>.</em>

**Comment by Fisherpon** [comment]
This is my first time using TumblThree, but I put my blogs in and left it to process overnight and nothing's happened. Not sure if I did something wrong, or Tumblr's updated their API and now it doesn't work. Does anyone know?

**Reply by MSWallack** [reply]
Did you remember to actually start the download? You wouldn't be the first person to queue a bunch of blogs to process but never start the download process.

**Reply by Fisherpon** [reply]
I did press the Download button, unless there's more to getting a download started than that. I do also have a Pihole on my network, so that may be getting in the way of things.

**Comment by Yorksbear1980** [comment]
Will you guys be releasing an extension to the program that allows the download of data from reblogme in the same way the program used to trawl newtumbl? I ask because I fear reblogme is going to meet the same fate as NewTumbl due to inappropriate content. Thanks.
